examples/formbot/data/stories.md

```diff
@@ -0,0 +1,98 @@
+## happy path
+* request_restaurant
+    - restaurant_form
+    - form{"name": "restaurant_form"}
+    - form{"name": null}
+* thank
+    - utter_noworries
+
+## chitchat once
+* request_restaurant
+    - restaurant_form
+    - form{"name": "restaurant_form"}
+* chitchat
+    - utter_chitchat
+    - restaurant_form
+    - form{"name": null}
+* thank
+    - utter_noworries
+
+## unhappy path
+* request_restaurant
+    - restaurant_form
+    - form{"name": "restaurant_form"}
+* chitchat
+    - utter_chitchat
+    - restaurant_form
+* chitchat
+    - utter_chitchat
+    - restaurant_form
+* chitchat
+    - utter_chitchat
+    - restaurant_form
+    - form{"name": null}
+* thank
+    - utter_noworries
+
+## unhappy path
+* request_restaurant
+    - restaurant_form
+    - form{"name": "restaurant_form"}
+* thank
+    - utter_chitchat
+    - restaurant_form
+    - form{"name": null}
+* thank
+    - utter_noworries
+
+## Generated Story -9155310465400161964 (example from interactive learning, "form: " will be excluded from training)
+* request_restaurant
+    - restaurant_form
+    - form{"name": "restaurant_form"}
+    - slot{"requested_slot": "cuisine"}
+* chitchat
+    - utter_chitchat
+    - restaurant_form
+    - slot{"requested_slot": "cuisine"}
+* form: inform{"cuisine": "mexican"}
```

Comment:
do you mean the `form:` prefix? I'd say yeah, because we want the user to know what was predicted by the form and what wasn't. Otherwise it's weird that training magically ignores some lines

Reply:
I prefer explicitly mark form lines with `form: ` rather than silently ignore them during training